proxy/envoy/testdata/all-cds.json.golden

```diff
@@ -74,6 +85,17 @@
      ]
     },
     {
+     "name": "out.httpsbin.default.svc.cluster.local|https",
```

Comment:
curious, why do we have so many copies of this cluster? is `all-cds` computing clusters for every node?